src/platform_impl/ios/monitor.rs

```diff
@@ -220,21 +218,22 @@ impl Inner {
         Some(refresh_rate_millihertz(self.uiscreen))
     }
 
-    pub fn video_modes(&self) -> impl Iterator<Item = RootVideoMode> {
+    pub fn video_modes(&self) -> impl Iterator<Item = VideoMode> {
         let mut modes = BTreeSet::new();
         unsafe {
             let available_modes: id = msg_send![self.uiscreen, availableModes];
             let available_mode_count: NSUInteger = msg_send![available_modes, count];
 
             for i in 0..available_mode_count {
                 let mode: id = msg_send![available_modes, objectAtIndex: i];
+                // Use Ord impl of RootVideoMode
                 modes.insert(RootVideoMode {
                     video_mode: VideoMode::retained_new(self.uiscreen, mode),
                 });
             }
         }
 
-        modes.into_iter()
+        modes.into_iter().map(|mode| mode.video_mode)
```

Comment:
That looks strange. Why do we even need `BTreeSet` in the first place here? To sort them somehow? I doubt other platforms sort anything here? And if sorted order is required can we do that on a higher level instead to force on all platforms instead?

Reply:
Windows does it for a reason:
https://github.com/rust-windowing/winit/blob/155f1f972046d77c9b0900db2d0500d89a7ada45/src/platform_impl/windows/monitor.rs#L228-L230

I suspect that iOS just followed along.

Reply:
Agree that it would be nicer to do the sorting on a higher level, though the deduplication probably makes sense to only do on Windows.

Reply:
Hm. Maybe that's fine the way it's then.